src/app/serverViewState.ts

```diff
@@ -68,7 +68,7 @@ export class ServerViewState {
     }
 
     getCurrentServer = () => {
-        log.debug('getCurrentServer');
+        log.silly('getCurrentServer');
```

Comment:
These logs kept just getting in the way as some of these methods are called very frequently (ie. on every click) so I changed the logging level on them.